### PR TITLE
add Footer Home, and refactor code

### PR DESCRIPTION
**Pull request by hsnice16**
Live: https://inspiring-kalam-dfc559.netlify.app/

Done with P0 and P1 components. Please review any give your feedback.

Thank you for reviewing.

### CODE REVIEW

**Review by VLeads — APPROVED** [review]
You have done the great work. keep going.

**Review by vishal804 — APPROVED** [review]

**Comment by vishal804** [comment]
Everything Looks Good Nice Work

File: pages/layout/css-grid.html [hunk 1]
```diff
@@ -350,7 +350,7 @@ <h3 class="my-1p5">On this Page</h3>
         </div>
       </footer>
 
-      <!-- Footer Ends -->
+      <!-- Footer Docs Ends -->
```
Comment (deekshithmd) [inline pages/layout/css-grid.html:353]: code and ui looks good to go...

**Review by deekshithmd — APPROVED** [review]

**Comment by hsnice16** [comment]
Thank you for reviewing.

**Pull request merged**